src/main/java/com/xiaomi/infra/pegasus/rpc/Cluster.java

```diff
@@ -55,17 +58,25 @@ public static Cluster createCluster(Properties config) throws IllegalArgumentExc
         Integer.parseInt(
             config.getProperty(
                 PEGASUS_PUSH_COUNTER_INTERVAL_SECS_KEY, PEGASUS_PUSH_COUNTER_INTERVAL_SECS_DEF));
+
+    boolean enableWriteLimit =
```

Comment:
You actually don't have to modify Cluster/ClusterManager and Table/TableHandler. Try out this way:

```java
class PegasusClient {
  public PegasusClient(Properties config) throws PException {
    this.config = config;
    this.cluster = Cluster.createCluster(config);
    this.enableWriteLimit = Boolean.parseBoolean(
            config.getProperty(PEGASUS_ENABLE_WRITE_LIMIT, PEGASUS_ENABLE_WRITE_LIMIT_DEF));
  }
  public bool isWriteLimitEnabled() { return enableWriteLimit; } 
}
```

```java
class PegasusTable {
  public PegasusTable(PegasusClient client, Table table) {
    this.writeLimiter = new WriteLimiter(client.isWriteLimitEnabled());
  }
}
```

Reply:
```
public PegasusClient(Properties config) throws PException {
    this.config = config;
    this.cluster = Cluster.createCluster(config);
    this.enableWriteLimit = Boolean.parseBoolean(
            config.getProperty(PEGASUS_ENABLE_WRITE_LIMIT, PEGASUS_ENABLE_WRITE_LIMIT_DEF));
  }
I think, the code placed here let our `config` init style is inconsistence and it looks like a little `disharmony`
```

Reply:
I don't think it really matters. If an option which is only used in PegasusTable, why need passing it to other classes?